tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "types": ["@types/node"],
     "noEmit": true,
-    "allowJs": true
+    "allowJs": true,
+    "allowImportingTsExtensions": true
```

Comment:
why is this needed?

Reply:
Without it, `yarn typecheck`'s `tsc` pass fails:

```plaintext
   ✖  nx run repo:typecheck
      > tsc
      
      tools/scripts/generate-configs.mts:18:8 - error TS5097: An import path can only end with a '.mts' extension when 'allowImportingTsExtensions' is enabled.
      
      18 } from './paths.mts';
                ~~~~~~~~~~~~~
      
      tools/scripts/generate-contributors.mts:10:27 - error TS5097: An import path can only end with a '.mts' extension when 'allowImportingTsExtensions' is enabled.
      
      10 import { REPO_ROOT } from './paths.mts';
                                   ~~~~~~~~~~~~~
      
      tools/scripts/generate-lib.mts:24:8 - error TS5097: An import path can only end with a '.mts' extension when 'allowImportingTsExtensions' is enabled.
      
      24 } from './paths.mts';
                ~~~~~~~~~~~~~
      
      tools/scripts/generate-sponsors.mts:6:34 - error TS5097: An import path can only end with a '.mts' extension when 'allowImportingTsExtensions' is enabled.
      
      6 import { PACKAGES_WEBSITE } from './paths.mts';
                                         ~~~~~~~~~~~~~
      
      
      Found 4 errors in 4 files.
      
      Errors  Files
           1  tools/scripts/generate-configs.mts:18
           1  tools/scripts/generate-contributors.mts:10
           1  tools/scripts/generate-lib.mts:24
           1  tools/scripts/generate-sponsors.mts:6
```

Reply:
Yeah this was an option set in the tsconfig of repo-tools that I needed to carry over for this reason